test/models/comment_test.rb

```diff
@@ -33,11 +33,11 @@ def setup
     assert_match "is too short (minimum is 2 characters)" , @comment_long_desc.errors[:description].join, "Minimum long error for description  is not found on comment.class"
   end
 
-  test "contains an user" do
-    assert_respond_to users(:esteban), @comment_valid.user, "Comment doesn't contain an author"
+  test "comment contains an user" do
```

Comment:
I believe the test should be "Comment respond to user" (which is that it needs to have the realatio with user)... So in assert_respopnd_to , the first param should a a comment object, not a user object